app/jobs/create_virtual_objects_job.rb

```diff
@@ -57,6 +57,7 @@ def perform(bulk_action_id, params)
         virtual_objects.reject! { |virtual_object| problematic_druids.include?(virtual_object[:parent_id]) }
       end
 
+      ActiveRecord::Base.clear_active_connections!
       errors = VirtualObjectsCreator.create(virtual_objects: virtual_objects)
       if errors.empty?
```

Comment:
You'll want to take a closer look at the code, but I think might want to place it after `VirtualObjectsCreator.create`, as that is the long-running process.

Reply:
I presume this is the long running line: https://github.com/sul-dlss/argo/blob/master/app/services/virtual_objects_creator.rb#L25

I wonder if instead we need to reconnect MySQL here during the polling?  :  https://github.com/sul-dlss/argo/blob/master/app/services/virtual_objects_creator.rb#L43

Reply:
We need to only deploy argo to stage to test this correct (i.e. no other projects need to be coordinate)?

Reply:
Yes, Argo only.

Reply:
If my understanding is correct, VirtualObjectsCreator just polls dor-services-app; it doesn't hit the db.

Reply:
Yeah, but I was thinking that if it sits in that method for a really long time running `redo` while it waits for the response, that is where the connection may get dropped.  But sure, I guess reconnecting once we exit the loop is fine too.

Reply:
move reconnect below job

Reply:
@peetucket Can you add a comment above this line so we can remember why we added it? This is great. Thanks!